src/components/ProfitAndLossDetailReport/ProfitAndLossDetailReport.tsx

```diff
@@ -176,15 +166,23 @@ export const ProfitAndLossDetailReport = ({
     [PnlDetailColumns.Description]: {
       id: PnlDetailColumns.Description,
       header: stringOverrides?.descriptionColumnHeader || 'Description',
-      cell: row => <Span ellipsis>{row.source?.displayDescription || row.account.accountSubtype.displayName || '-'}</Span>,
+      cell: row => (
+        <Text
+          as='span'
+          withTooltip={TextUseTooltip.whenTruncated}
```

Comment:
Note that I'm unfortunately going back to using the old `<Text />` component here because it supports this tooltip functionality which, after trying it with real world data, is a must-have. 

I tried porting over tooltip functionality into the new text components but there is a non-trivial refactor to be done there, punting for now.